css-highlight-api-1/Overview.bs

```diff
@@ -668,15 +668,19 @@ method must return the result of running these steps:
 	* <var>x</var> is greater than the <a>viewport</a> width excluding the size of a rendered scroll bar (if any)
 	* <var>y</var> is greater than the <a>viewport</a> height excluding the size of a rendered scroll bar (if any)
 1. Otherwise, return a [=sequence=] of [=custom highlights=] given by ordering the highlights contained in this {{HighlightRegistry}} in descending order of [=priority=],
-	excluding the highlights without at least one [=range=] <var>range</var> that satisfies the following constraints:
-	1. The coordinates <var>x</var>,<var>y</var> fall inside at least one of the {{DOMRect}}s returned by calling {{Range/getClientRects()}} on <var>range</var>.
+	including only those highlights that contain at least one {{AbstractRange}} <var>abstractRange</var> that satisfies the following:
+
+	* Let <var>range</var> be a {{Range}} object whose [=start node=] and [=end node=] are set to <var>abstractRange</var>'s [=start node=] and [=end node=] respectively,
```

Comment:
This is basically a workaround for the fact that StaticRange doesn't support getClientRects. This is fine for now, but should we also file an issue to support getClientRects (and presumably getBoundingClientRect) on AbstractRange so that it works for all range types?

Reply:
Sounds good, I filed an issue here https://github.com/w3c/csswg-drafts/issues/11813